plugins/reporters/spdx/src/main/kotlin/Extensions.kt

```diff
@@ -70,10 +70,14 @@ private fun Hash.toSpdxChecksum(): SpdxChecksum? =
         // The SPDX checksum algorithm names are simple and assumed to be part of ORT's HashAlgorithm aliases.
         .find { it.name in algorithm.aliases }
         ?.let {
-            SpdxChecksum(
-                algorithm = it,
-                checksumValue = value
-            )
+            runCatching {
+                SpdxChecksum(
+                    algorithm = it,
+                    checksumValue = value
```

Comment:
IMO we should simply call `value.lowercase()` here instead. Exceptions should probably be caught on a higher level anyway.